test/Microsoft.ML.TestFramework/BaseTestBaseline.cs

```diff
@@ -96,6 +97,7 @@ protected override void Initialize()
             _baselineCommonDir = Path.Combine(baselineRootDir, "Common");
             _baselineBuildStringDir = Path.Combine(baselineRootDir, BuildString);
             _baselineConfigDirs = GetConfigurationDirs();
+            _usedSpecificBaselineConfigs = new HashSet<string>();
```

Comment:
>_usedSpecificBaselineConfigs [](start = 12, length = 28)

Do you have the ability to log configuration SingleDebug and SingleRelease?

Reply:
Unfortunately, SingleDebug and SingleRelease are treated differently right now. 

@mstfbl : After this PR is fixed and merged, can you please create a new PR to merge the logic for handing Debug and Release with the same logic as GetConfigurationDirs()? (Not high priority)

---
In reply to: [416078739](https://github.com/dotnet/machinelearning/pull/5060#discussion_r416078739) [](ancestors = 416078739)